lib/rules/prop-types.js

```diff
@@ -466,11 +466,22 @@ module.exports = {
      * @return {Object} The representation of the declaration, empty object means
      *    the property is declared without the need for further analysis.
      */
-    function buildTypeAnnotationDeclarationTypes(annotation) {
+    function buildTypeAnnotationDeclarationTypes(annotation, seen) {
+      if (seen === void 0) {
```

Comment:
`typeof seen === 'undefined'` please :-) either way, there's no need for `void 0` in code that only runs in a node environment.

Reply:
done